<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardTheme</class>
 <widget class="QDialog" name="KeyboardTheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard Themes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="keyboardThemes"/>
     </item>
     <item>
      <widget class="QPushButton" name="newThemeButton">
       <property name="text">
        <string>New Theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteThemeButton">
       <property name="text">
        <string>Delete Theme</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="KeyboardMap">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Function</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mapped To</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Set Mapping</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QKeySequenceEdit" name="keySequenceEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="KeyboardFunctionList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setKeyboardMap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="message">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>505</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newThemeButton</sender>
   <signal>clicked()</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>addTheme()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteThemeButton</sender>
   <signal>clicked()</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>deleteTheme()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keyboardThemes</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>themeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keySequenceEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>truncateShortcut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setKeyboardMap</sender>
   <signal>clicked()</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>setKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>680</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>KeyboardMap</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>KeyboardTheme</receiver>
   <slot>populateKeySequence(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTheme()</slot>
  <slot>deleteTheme()</slot>
  <slot>themeChanged(int)</slot>
  <slot>truncateShortcut()</slot>
  <slot>populateKeySequence(QTableWidgetItem*)</slot>
  <slot>setKey()</slot>
 </slots>
</ui>
